src/services/walletService.ts

```diff
@@ -7,6 +7,7 @@ import { pvtKeyEncryptionManager } from '@/actions/pvtKeyEncryptMgmt'
 import base58 from 'bs58'
 
 const customRpcUrl = process.env.NEXT_PUBLIC_SOLANA_RPC || ''
+const coinGeckoUrl = process.env.NEXT_PUBLIC_COIN_GECKO_API || ''
 
```

Comment:
You can give a default url instead of the null string on  or

Reply:
No, let it be keep it an empty string